src/main/java/com/example/demo/service/DemoService.java

```diff
@@ -19,10 +28,30 @@ public DemoService(SimpMessagingTemplate simpMessagingTemplate) {
         this.simpMessagingTemplate = simpMessagingTemplate;
     }
 
-    @Scheduled(cron = "*/5 * * * * *")
-    public void performTask() {
+    @Scheduled(cron = "0 * * * * *")
+    public void performTask() throws IOException {
         Instant now = Instant.now();
         logger.info("Scheduled task performed at {} (ISO 8601 date and time format)", now);
+
+        OkHttpClient client = new OkHttpClient();
+        Request request = new Request.Builder()
+                .url("https://jsonplaceholder.typicode.com/users/1")
+                .build();
+        Response response = client.newCall(request).execute();
+        String responseBody = response.body().string();
+        logger.info("Response: {}", responseBody);
+
+        EnvironmentVariableCredentialsProvider credentialsProvider = new EnvironmentVariableCredentialsProvider();
+        AmazonS3 s3client = AmazonS3ClientBuilder.standard()
+                .withRegion(Regions.US_EAST_1)
+                .withCredentials(credentialsProvider)
+                .build();
+
+        ObjectListing objectListing = s3client.listObjects("sea-1074-amazon-codeguru-poc");

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

This code uses an outdated API. [ListObjectsV2 is the revised List Objects API, and we recommend you use this revised API for new application developments](https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html).